tutorial.md

```diff
@@ -163,7 +163,7 @@ Description=OCI-based implementation of Kubernetes Container Runtime Interface
 Documentation=https://github.com/kubernetes-incubator/cri-o
 
 [Service]
-ExecStart=/usr/local/bin/crio --log-level debug
+ExecStart=/usr/local/bin/crio
```

Comment:
It would be nice to show an example in crio.conf how to set up the debug level.  Also based on the registry.conf discussions this morning, does the registries option at line 150 still hold?

Reply:
That is fine, for 1.10 and below we need to add the default registries in `/etc/crio/crio.conf`.
Added an "optional" to show how we can change the log_level in crio.conf.